app/assets/stylesheets/layout/page_specific/_article_page.scss

```diff
@@ -109,3 +109,65 @@
     @include column-shift(3);
   }
 }
+
+// BackToTop link
+$btt-link-height: 42px;
+
+.article-3col-main__btt-link {
+  position: fixed;
+  top: auto;
+  bottom: 0 - ($btt-link-height + 6px);
+  right: 0;
+  margin: 0;
+  padding: $baseline-unit*1.5 10px $baseline-unit*1.5 40px;
+  border-radius: $baseline-unit*1.5 0 0 0;
+  white-space: nowrap;
+  color: $color-white;
+  font-weight: bold;
+  text-decoration: none;
+  background-color: $color-green-secondary;
+  z-index: 1;
+  height: $btt-link-height;
+  transition: bottom, 0.4s;
+
+  &:focus {
+    background-color: $color-green-secondary;
+  }
+
+  &:visited {
+    color: $color-white;
+  }
+}
+
+.article-3col-main__btt-link--shown {
+  position: fixed;
+  top: auto;
+  bottom: 0;
+}
+
+.article-3col-main__btt-link__arrow {
+  width: 20px;
+  height: 12.1572px;
+  position: absolute;
+  left: 10px;
+  top: 50%;
+  transform: translateY(-50%);
+  fill: $color-white;
+}
+
+.article-3col-main__btt-link:hover,
+.article-3col-main__btt-link:focus {
+  text-decoration: underline;
+}
+
+.footer--btt_link {
```

Comment:
This doesn't really fit with our BEM conventions. I'd suggest that we put the 'btt-link' in a separate file and then have 'footer' or 'article' as the modifier?